specification/desktopvirtualization/resource-manager/readme.go.md

```diff
@@ -12,14 +12,15 @@ go:
 ### Go multi-api
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2019-01-23-preview
+  - tag: package-2019-12-10-preview

```

Comment:
Usually we do not remove the old tag, just append a new tag to batch. Do we have some particular reason of doing this?

Reply:
we want to start off with the latest version. The earlier ones were before public preview.

Reply:
adding them.